src/Nordea/Components/TextInput.elm

```diff
@@ -184,77 +178,41 @@ onEnterPress msg =
 
 view : List (Attribute msg) -> TextInput msg -> Html msg
 view attributes (TextInput config) =
-    if Maybe.isJust config.onClearInput then
```

Comment:
Hmm, think I changed my mind a little 🙈 What do you think about keeping the setup as it was, but keep the changes you made to the input to this component? (change from `Maybe msg` to `Bool`)

Instead of having `, config.onClearInput |> Maybe.map onClick` as the onClick to the button, we can have `, config.onInput |> Maybe.map (\onInput -> onClick (onInput ""))`. That way, the consumers of this component does not need to setup a new msg and update logic to use it.

Reply:
Yes, webkit thingy is taking things out of the Elm a lot. Setting "" is good